android/cli/commands/_buildModule.js

```diff
@@ -1153,7 +1153,7 @@ AndroidModuleBuilder.prototype.compileJsClosure = function (next) {
 		}),
 		closureJarFile = path.join(this.platformPath, 'lib', 'closure-compiler.jar');
 
-	jsFilesToEncrypt.forEach(function (file) {
+	async.each(jsFilesToEncrypt, function(file, callback) {
```

Comment:
This should probably be an `async.eachLimit(jsFilesToEncrypt, 5, function (file, callback) {` so that it doesn't spin up hundreds of Java instances.